zipline/data/data_portal.py

```diff
@@ -168,13 +168,33 @@ def __init__(self,
 
         self._first_trading_session = first_trading_day
 
-        _last_sessions = [r.last_available_dt
-                          for r in [equity_daily_reader, future_daily_reader]
-                          if r is not None]
-        if _last_sessions:
-            self._last_trading_session = min(_last_sessions)
+        if last_available_session:
+            self._last_trading_session = last_available_session
```

Comment:
What would you think of aligning the names here (i.e. `self._last_trading_session` -> `self._last_available_session`)? Seems like that maybe have been a source of confusion here